api/cpp/Cargo.toml

```
@@ -45,12 +45,13 @@ renderer-software = ["i-slint-backend-selector/renderer-software"]
 gettext = ["i-slint-core/gettext-rs"]
 accessibility = ["i-slint-backend-selector/accessibility"]
 system-testing = ["i-slint-backend-selector/system-testing"]
+raw-window-handle-06 = ["i-slint-core/raw-window-handle-06", "i-slint-backend-selector/raw-window-handle-06"]
 
 std = ["i-slint-core/default", "i-slint-core/image-default-formats", "i-slint-backend-selector"]
```

So this is what i mean:

```suggestion
std = ["i-slint-core/default", "i-slint-core/image-default-formats", "i-slint-backend-selector/raw-window-handle-06", "i-slint-core/raw-window-handle-06"]
```

If the `std` feature is there, enable the raw window handle. No need for a "raw-window-handle-06" feature in cmake.

---

Done, that works now!